ray-operator/config/samples/ray-cluster-alb-ingress.yaml

```diff
@@ -3,7 +3,10 @@ kind: Ingress
 metadata:
   name: ray-cluster-ingress
   annotations:
-    alb.ingress.kubernetes.io/scheme: internet-facing
+    # WARNING: Do not expose this ALB publicly without additional authentication/authorization.
+    # The Ray Dashboard provides read and write access to the cluster. Anyone with access to the
+    # ALB can launch arbitrary code execution on the Ray Cluster.
+    alb.ingress.kubernetes.io/scheme: internal
```

Comment:
Does [this doc](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/ingress.html#aws-application-load-balancer-alb-ingress-support-on-aws-eks) still work after the change? In addition, would you mind explaining what's the difference from security perspective between `internal` and `internal-facing`?

Reply:
The difference between `internal` and `**internet**-facing` is that `internal` requires access from within the VPC, whereas `internet-facing` is exposed on the public internet. It is unsafe to publicly expose the Ray Dashboard on the internet because that allows anyone to submit Ray Jobs (and thus execute arbitrary code) on the cluster.

`internal` requires a VPC or similar technology to reach, but is _more safe by default_.

Reply:
Thanks for the explanation! Based on the explanation, [this doc](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/ingress.html#aws-application-load-balancer-alb-ingress-support-on-aws-eks) seems to no longer work without modification.

Reply:
I'm going to update that as well. It will technically get the ALB up and running, but will not give immediate access.

Reply:
Can we cherry-pick the doc update to Ray 2.7.0? I guess the doc uses the YAML file in the KubeRay master branch. If we merge this PR at this moment, users may have issues when they try the doc on the Ray website.